mousestyles/distribution/test/test_kolmogorov_test.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function, absolute_import, division
+
+import numpy as np
+from scipy import stats
+from mousestyles.distribution.kolmogorov_test import perform_kstest
+from mousestyles.distribution.kolmogorov_test import get_travel_distances
+
+
+def test_get_travel_distances():
+    res = get_travel_distances(strain=0, mouse=0, day=0)
+    assert type(res) is np.ndarray
+    assert len(res) == 38845
+    assert np.allclose(res[:3], np.array([1.00648944, 1.02094319, 1.0178885]))
+
+
+def test_perform_kstest():
+    res = perform_kstest(strain=0, mouse=0, day=0,
+                         distribution=stats.pareto, verbose=False)
+    assert type(res) is np.ndarray
+    assert res.shape == (3,)
+    assert np.allclose(res, np.array([3.67593246, 0.62795748, 0.37224205]))
```

Comment:
I don't think this is a good test to include because the values [3.67593246, 0.62795748, 0.37224205] don't really mean anything a priori. You should just remove this.